lib-injection/host_inject.rb

```diff
@@ -83,11 +83,14 @@ def in_bundle?
           Bundler::SharedHelpers.in_bundle?
         end
 
+        # CRuby does *not* break API or ABI compatibility in TEENY version bumps.
+        # The ABI always sets the TEENY version to zero to reflect that: {MAJOR}.{MINOR}.0.
+        # https://www.ruby-lang.org/en/news/2013/12/21/ruby-version-policy-changes-with-2-1-0/
         def runtime_supported?
-          major, minor, = RUBY_VERSION.split('.')
+          major, minor, = RUBY_VERSION.split('.') # {MAJOR}_{MINOR}_{TEENY}
           ruby_api_version = "#{major}.#{minor}.0"
```

Comment:
What is the purpose of `.0` here? If only two components are used by the logic why not just have two components here?